imaspy/test/test_nc_autofill.py

```diff
@@ -1,11 +1,56 @@
 from imaspy.db_entry import DBEntry
+from imaspy.exception import InvalidNetCDFEntry
 from imaspy.test.test_helpers import compare_children, fill_consistent
+import re
+import pytest
+import netCDF4
+from packaging import version
 
 
-def test_nc_latest_dd_autofill_put_get(ids_name, tmp_path):
+def test_nc_latest_dd_autofill_put_get_skip_complex(ids_name, tmp_path):
```

Comment:
discussed with @DaanVanVugt to separately test the `skip_complex=False/True` pathway. 

Q: Are all three of these tests useful to keep here?

Reply:
For me three tests is fine: they're not overly big or complicated, and it's good that we can test the compatibility with older netCDF libraries